resources/testing/documents/SFI23/data/SFI23-ffc-pay-statement-data.sql.bak

```diff
@@ -0,0 +1,20 @@
+TRUNCATE TABLE public.organisations RESTART IDENTITY CASCADE;

```

Comment:
Just needs the backup file removing if it is not needed for public use

Reply:
good spot!